ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -187,52 +187,60 @@ tests:
   container:
     from: src
 - as: e2e-aws
-  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(azure|baremetal|gcp|libvirt|openstack|ovirt|vsphere)/)
+  skip_if_only_changed: (^(docs|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(alibabacloud|azure|baremetal|gcp|ibmcloud|libvirt|openstack|ovirt|vsphere)/)
```

Comment:
We should probably update all of these (non-UPI) jobs to ignore the `upi` directory as well as `docs` and `hack`.